apps/web-giddh/src/app/settings/permissions/form/form.component.html

```diff
@@ -1,143 +1,219 @@
-                    <div class="row mrB1"
-                        *ngFor="let item of permissionForm.get('allowedCidrs')['controls']; let i = index; let l = last"
-                        [formGroupName]="i">
-                        <div class="col-xs-11">
-                            <input class="form-control " name="item_{{i}}" formControlName="range" type="text"
-                                placeholder="192.168.1.0/24" />
-                        </div>
-                        <div class="pull-left lh34">
-                            <i *ngIf="!l" (click)="delRow('allowedCidrs', i, $event)"
-                                class="fa fa-trash-o cursor-pointer"></i>
-                            <i *ngIf="l" (click)="addNewRow('allowedCidrs', item, $event)"
-                                class="fa fa-plus cursor-pointer"></i>
+    <!--  -->
+    </div>
+    <div class="mb-4">
+    <p class="mb-15">
+        <input type="checkbox" (change)="show = !show" ng-model="show" />
+        Payment Permissions</p>
+    <div class="row container-fluid permission-main" *ngIf="show">
+        <div class="clearfix d-flex permission-row">
+            <div class="col-sm-6">
+                <div class="d-flex justify-content-between align-items-center">
+                    <label>Bank Account</label>
+                    <div class="relative d-inline-block transactions-limit-box">
+                        <a class="text-gray text-gray pr-1 d-inline-block" href="javascript:void(0)">
+                        <i class="icon-info"></i></a>
+                        <div class="transactions-popup">
+                            <p class="bold mb-1">Bank Account</p>
+                            <p class="font-12 text-gray mb-1"> Aenean non accumsan ante. Duis et risus accumsan sem tempus porta nec sit amet est. Sed ut euismod quam. Suspendisse potenti. Aliquam fringilla orci tincidunt, ullamcorper erat in, malesuada metus.</p>
+                            <a class="btn-link bold font-14" href="javascript:void(0)">More Details</a>
                         </div>
                     </div>
-                </section>
-
-                <section formArrayName="allowedIps" class="iprange container-fluid"
-                    *ngIf="permissionForm.get('ipOptions').value === 'ip_address'">
-                    <div class="row mrB1"
-                        *ngFor="let item of permissionForm.get('allowedIps')['controls']; let i = index; let l = last"
-                        [formGroupName]="i">
-                        <div class="col-xs-11">
-                            <input class="form-control" (keypress)="allowOnlyNumbersAndDot($event)" name="item_{{i}}"
-                                formControlName="range" type="text" placeholder="192.168.1.0" />
+                </div>
+                <sh-select [options]="approvalName" class="text-left" name="selectedBankName" required [placeholder]="'Select Bank'" [multiple]="false">
```

Comment:
Use normal placeholder attribute for static text

Reply:
placeholder changed [placeholder]="'Select Bank'" to placeholder="Select Bank"